bundles/org.openhab.binding.nuvo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -6,7 +6,7 @@
 
 	<!-- Nuvo Whole House Amplifier Thing -->
 	<thing-type id="amplifier">
-		<label>Nuvo Whole House Amplifier</label>
+		<label>Grand Concerto or Essentia G Whole House Amplifier</label>
```

Comment:
Labels are kept short intentionally:

See: https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

> The label should be short so that for most UIs it does not spread across multiple lines. Guideline is 2-3 words with up to 25 chars.

Maybe it also works by updating the description instead?

Reply:
It will come up as a binding match based on text in the description but not a thing match. Anyway I moved the new text to the description tag.